<?xml version="1.0" encoding="UTF-8"?>
<!--
    Copyright 2014-2015 Vanniktech - Niklas Baudy

    This file is part of SpeedReader.

    SpeedReader is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 2 of the License, or
    (at your option) any later version.

    SpeedReader is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with SpeedReader. If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpeedReader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="startPage">
       <layout class="QHBoxLayout" name="startPageLayout" stretch="1,2">
        <item>
         <layout class="QHBoxLayout" name="startPageWidget">
          <item>
           <widget class="QScrollArea" name="navigationDrawerScrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="navigationDrawerScrollAreaWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>120</width>
               <height>449</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="pasteWidget" native="true">
            <layout class="QVBoxLayout" name="pasteWidgetLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit">
               <property name="placeholderText" stdset="0">
                <string>Type or paste your text here to SpeedRead it</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="buttonLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QPushButton" name="speedreadButton">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: S&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>SpeedRead</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="continueButton">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Continue</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="googlePlayStoreLabel">
               <property name="text">
                <string>Google Play Store</string>
               </property>
               <property name="textFormat">
                <enum>Qt::RichText</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="donateLabel">
               <property name="text">
                <string>Donate</string>
               </property>
               <property name="textFormat">
                <enum>Qt::RichText</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="rssWidget" native="true">
            <layout class="QVBoxLayout" name="rssWidgetLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="rssListWidget">
               <property name="iconSize">
                <size>
                 <width>80</width>
                 <height>80</height>
                </size>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="statusWidget" native="true">
            <layout class="QVBoxLayout" name="statusWidgetLayout">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="segmentsStatusLabel"/>
             </item>
             <item>
              <widget class="QLabel" name="stopWordsStatusLabel"/>
             </item>
             <item>
              <widget class="QLabel" name="estimatedRemainingSpeedReadingTimeLabel"/>
             </item>
             <item>
              <widget class="QTextEdit" name="textEdit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="speedreadPage">
       <layout class="QHBoxLayout" name="speedreadPageLayout">
        <item>
         <widget class="QWidget" name="speedreadWidget" native="true">
          <layout class="QVBoxLayout" name="speedreadWidgetLayout">
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="stopCancelButtonLayout">
             <item>
              <widget class="QPushButton" name="stopButton">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: S&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancelButton">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shortcut: C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSpeedReader"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutSpeedReader"/>
    <addaction name="separator"/>
    <addaction name="actionDonate"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionForkMeOnGitHub"/>
    <addaction name="separator"/>
    <addaction name="actionReportABug"/>
    <addaction name="separator"/>
    <addaction name="actionSendFeedback"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Languages</string>
    </property>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionSpeedReader">
   <property name="text">
    <string>SpeedReader</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAboutSpeedReader">
   <property name="text">
    <string>SpeedReader</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionReportABug">
   <property name="text">
    <string>Report a bug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionSendFeedback">
   <property name="text">
    <string>Send feedback</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionForkMeOnGitHub">
   <property name="text">
    <string>Fork me on GitHub</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
